src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -211,6 +332,83 @@ private IFeatureFilterMetadata GetFeatureFilterMetadata(string filterName)
             return filter;
         }
 
+        private IFeatureVariantAssignerMetadata GetFeatureAssignerMetadata(string assignerName)
+        {
+            const string assignerSuffix = "assigner";
+
+            IFeatureVariantAssignerMetadata assigner = _assignerMetadataCache.GetOrAdd(
+                assignerName,
+                (_) => {
+
+                    IEnumerable<IFeatureVariantAssignerMetadata> matchingAssigners = _variantAssigners.Where(a =>
+                    {
+                        Type filterType = a.GetType();
+
+                        string name = ((AssignerAliasAttribute)Attribute.GetCustomAttribute(filterType, typeof(AssignerAliasAttribute)))?.Alias;
+
+                        if (name == null)
+                        {
+                            name = GetTrimmedName(a.GetType(), assignerSuffix);
```

Comment:
If I have an assigner type `FooAssigner`, looks like we won't match it if have I `"Assigner": "FooAssigner"` in my config unless I use an alias on my assigner type or have `"Assigner": "Foo"` in my config. Is this intended?

Reply:
An [issue ](https://github.com/microsoft/FeatureManagement-Dotnet/issues/12)that was never addressed. We can fix it.

Reply:
Added support for that and added a test.